# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dosfstools/reinstall.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Fixed removal of existing symlinks before creation.

  - Rene Rebe <rene@exactcode.de>

--- ./dosfsck/Makefile.vanilla	2005-01-22 00:52:02.398320408 +0100
+++ ./dosfsck/Makefile	2005-01-22 00:52:13.061699328 +0100
@@ -18,6 +18,7 @@
 	ln -s dosfsck $(SBINDIR)/fsck.msdos
 	ln -s dosfsck $(SBINDIR)/fsck.vfat
 	rm -f $(MANDIR)/fsck.msdos.8
+	rm -f $(MANDIR)/fsck.vfat.8
 	ln -s dosfsck.8 $(MANDIR)/fsck.msdos.8
 	ln -s dosfsck.8 $(MANDIR)/fsck.vfat.8
 
--- ./mkdosfs/Makefile.vanilla	2005-01-22 00:51:20.139744688 +0100
+++ ./mkdosfs/Makefile	2005-01-22 00:51:39.801755608 +0100
@@ -18,6 +18,7 @@
 	ln -s mkdosfs $(SBINDIR)/mkfs.msdos
 	ln -s mkdosfs $(SBINDIR)/mkfs.vfat
 	rm -f $(MANDIR)/mkfs.msdos.8
+	rm -f $(MANDIR)/mkfs.vfat.8
 	ln -s mkdosfs.8 $(MANDIR)/mkfs.msdos.8
 	ln -s mkdosfs.8 $(MANDIR)/mkfs.vfat.8
 
